### ISSUE TITLE
ClearMatrix

### ISSUE THREAD

**Issue by mcsarge** [easybotics/node-red-contrib-easybotics-led-matrix#42]
Is there any way to issue a ClearMatrix command or have it added as a node? As is, I can only really display 1 page. If I could clear off the page, I could simulate several pages of data.

**Comment by RaceTrackr** [comment]
Agreed. Lots of strange behaviour with the panel if trying to clear via other methods (drawing polygons for example).

**Comment by clowrey** [comment]
If you'd like to add it and make a pull request after thoroughly testing I'm open to that possibility 🙂

**Comment by RaceTrackr** [comment]
> If you'd like to add it and make a pull request after thoroughly testing I'm open to that possibility 🙂

I have been working on a local fork for a while and keep hitting big stumbling blocks with unexpected behaviour. No errors, just weird quirks. I have programmed a fair amount of these panels with ESPs and just can't wrap my head around said behaviour!

**Comment by clowrey** [comment]
It's been awhile since I actively worked with them. Still have a display on the wall here functional after 3 years continuously running with the rpi + node red, so they are reliable once going ;).

**Comment by RaceTrackr** [comment]
> 

Oh for sure. It's a much better solution than using ESPs (hence moving across).

The biggest issue I seem to be facing is removing something from the buffer. I write 'hello' then attempt to fill whole panel with white, then fill whole panel with black (to force a clear), I cannot then write something new using another node on top of the black.

I can write a new string to the text node which updates it, but I cannot remove that text node once it's done it's first message. If I send a blank payload or "" or " ", I get the object or a '0' displayed respectively. 

The panel is battery powered and only needs to display for 2 minutes at a time. I'm trying to find a way to clear the display in a sensible way after 2 minutes that allows to be rewritten to at will.

Honestly I'm about 3 failures away to writing the whole program in python!

**Comment by clowrey** [comment]
Yeah I understand the frustration! We had allot of issues getting these things to work initially too, wish I had the bandwidth to help..
